### PR TITLE
Adds Missing Const Specifiers to Member Functions

### PR DESCRIPTION
**Pull request by economon**
## Proposed Changes
*Give a brief overview of your contribution here in a few sentences.*
 
Adds const specifier to member functions and also const to some function input args where applicable.

## Related Work
*Resolve any issues (bug fix or feature request), note any related PRs, or mention interactions with the work of others, if any.*

N/A

## PR Checklist
*Put an X by all that apply. You can fill this out after submitting the PR. If you have any questions, don't hesitate to ask! We want to help. These are a guide for you to know what the reviewers will be looking for in your contribution.*

- [x] I am submitting my contribution to the develop branch.
- [x] My contribution generates no new compiler warnings (try with the '-Wall -Wextra -Wno-unused-parameter -Wno-empty-body' compiler flags).
- [x] My contribution is commented and consistent with SU2 style.

### CODE REVIEW

File: Common/include/geometry/CGeometry.hpp [hunk 1]
```diff
@@ -1585,7 +1585,7 @@ class CGeometry {
    * \brief Compute and store the volume of the elements.
    * \param[in] config - Problem configuration.
    */
-  void SetElemVolume(CConfig *config);
+  void SetElemVolume(CConfig *config) const;
```
Comment (pcarruscag) [inline Common/include/geometry/CGeometry.hpp:1589]: This method is not const, it changes the geometry.

File: Common/include/geometry/CGeometry.hpp [hunk 2]
```diff
@@ -867,27 +867,27 @@ class CGeometry {
    * \param[in] config - Definition of the particular problem.
    * \param[in] print - Display information on screen.
    */
-  void SetRotationalVelocity(CConfig *config, bool print = false);
+  void SetRotationalVelocity(CConfig *config, bool print = false) const;
```
Comment (pcarruscag) [inline Common/include/geometry/CGeometry.hpp:870]: neither is this one

File: Common/include/geometry/CGeometry.hpp [hunk 3]
```diff
@@ -522,14 +522,14 @@ class CGeometry {
...
+  inline void SetMarker_Tag(unsigned short val_marker, string val_index) const { Tag_to_Marker[val_marker] = std::move(val_index); }
 
   /*!
    * \brief Set the number of boundary elements.
    * \param[in] val_marker - Marker of the boundary.
    * \param[in] val_nelem_bound - Number of boundary elements.
    */
-  inline void SetnElem_Bound(unsigned short val_marker, unsigned long val_nelem_bound) {
+  inline void SetnElem_Bound(unsigned short val_marker, unsigned long val_nelem_bound) const {
     nElem_Bound[val_marker]= val_nelem_bound;
```
Comment (pcarruscag) [inline Common/include/geometry/CGeometry.hpp:533]: clearly not const, even if the compiler allows it.

File: Common/include/geometry/CGeometry.hpp [hunk 4]
```diff
@@ -867,27 +867,27 @@ class CGeometry {
...
-  void SetTranslationalVelocity(CConfig *config, bool print = false);
+  void SetTranslationalVelocity(CConfig *config, bool print = false) const;
 
   /*!
    * \brief Set the grid velocity via finite differencing at each node.
    * \param[in] config - Definition of the particular problem.
    * \param[in] iter - Current physical time step.
    */
-  void SetGridVelocity(CConfig *config, unsigned long iter);
+  void SetGridVelocity(CConfig *config, unsigned long iter) const;
```
Comment (pcarruscag) [inline Common/include/geometry/CGeometry.hpp:890]: what should be const in all of these is the config

**Comment by economon** [comment]
As you probably guessed, these changes were automated. I have made a pass manually and removed const from setter methods where it doesn't make sense.

**Pull request merged**